lib/ota/portable/nordic/nrf52840-dk/aws_ota_pal.c

```diff
@@ -189,21 +192,12 @@ OTA_Err_t prvPAL_CreateFileForRx( OTA_FileContext_t * const C )
     if (xStatus == kOTA_Err_None){
         /* Erase the required memory */
         OTA_LOG_L1( "[%s] Erasing the flash memory \n", OTA_METHOD_NAME );
-        xStatus = prvErasePages(otapalSECOND_BANK_START, otapalSECOND_BANK_START + otapalDESCRIPTOR_SIZE + C->ulFileSize);
+        xStatus = prvErasePages(otapalSECOND_BANK_START, otapalSECOND_BANK_SIZE);
```

Comment:
Thank you for spotting it out! We need to do it because:
- We need to erase all trace of previous firmware on it
- The tests don't pass the size so it is not going to pass otherwise 
- Other ports do it
After discussion that is what we came up internally. We chose consistency.